src/ui-shell/sidenav/sidenav-item.interface.ts

```diff
@@ -0,0 +1,8 @@
+export interface SideNavItemInterface {
```

Comment:
It may be possible to have `HeaderItemInterface` and `SideNavItemIterface` share an interface ... or at least have one extend the other ... Really they're both "menu items" and barring `isSubMenu` (which is a bit of a weird case anyway) they're already 99% identical 🤔 

They should definitely share a base interface at least though